.github/workflows/test-action.yml

```
@@ -1,397 +1,340 @@
+          repo: ${{ env.REPO }}
+          workflow_id: ${{ env.WORKFLOW_ID }}
+          ref: ${{ env.REF }}
+          inputs: '{"Timeout": "5"}'
+          wait_interval: ${{ env.WAIT_INTERVAL }}
+          timeout: ${{ env.TIMEOUT }}
+          action: "trigger-and-wait"
+          debug: ${{ env.DEBUG }}
 
-      - name: '[2] (token) Verify Outputs'
+      - name: "Validate"
         run: |
```

actionlint (1.7.7)

property "run_conclusion" is not defined in object type {}

(expression)